Mathlib/NumberTheory/LSeries/PrimesInAP.lean

```diff
@@ -248,6 +288,193 @@ lemma LSeries_residueClass_eq (ha : IsUnit a) {s : ℂ} (hs : 1 < s.re) :
   simp only [Pi.smul_apply, residueClass_apply ha, smul_eq_mul, ← mul_assoc,
     mul_inv_cancel_of_invertible, one_mul, Finset.sum_apply, Pi.mul_apply]
 
+variable (a)
+
+open Classical in
+/-- The auxiliary function used, e.g., with the Wiener-Ikehara Theorem to prove
+Dirichlet's Theorem. On `re s > 1`, it agrees with the L-series of the von Mangoldt
+function restricted to the residue class `a : ZMod q` minus the principal part
+`(q.totient)⁻¹/(s-1)` of the pole at `s = 1`; see `DirichletsThm.auxFun_prop`. -/
```

Comment:
It seems `auxFun_prop` is not part of this PR.

Reply:
I changed `auxFun_prop` to `eqOn_auxFun` at some point, but forgot to update the docstring. Will fix.

Reply:
I found a few other places where a change in namespace was not reflected in the docstrings.